server.js

```diff
@@ -0,0 +1,28 @@
+/*
+*  Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+*  See LICENSE in the source repository root for complete license information.
+*/
+
+var express = require('express');
+var app = express();
+var morgan = require('morgan');
+var path = require('path');
+
+// Initialize variables.
+var port = 1530 // process.env.PORT || 8080;
+
+// Configure morgan module to log all requests.
+app.use(morgan('dev'));
+
+// Set the front-end folder to serve public assets.
+app.use(express.static(__dirname + '/public'));
```

Comment:
Currently, in the example, there are two masl.js. Don't we want index.html to just reference public/msal.js?
This would make the sample work as well when not started from node.js?